script/BatchRegisterValidator.s.sol

```diff
@@ -54,14 +60,37 @@ contract BatchRegisterValidator is Script {
         uint256 vtAmount = vm.promptUint("Enter the VT amount per validator (28 is minimum):");
```

Comment:
just something I forgot to push on my last commit: remove the `:` from the end, promtUint already prints one.